docs/how-to-build.md

```diff
@@ -88,17 +93,11 @@ This software is licensed under the Apache License version 2."
    with specific module name, like:
 
     ```shell
-    ./gradlew spark-connector:spark-runtime-3.4:build -PscalaVersion=2.12
+    ./gradlew spark-connector:spark-connector-runtime:build
     ```
 
    This creates `gravitino-spark-connector-runtime-{sparkVersion}_{scalaVersion}-{version}.jar`
-   under the `spark-connector/v3.4/spark-runtime/build/libs` directory. You could replace `3.4` with 
-   `3.3` or `3.5` to specify different Spark versions, replace `2.12` with `2.13` for different Scala 
-   version. The default Scala version is `2.12` if not specifying `-PscalaVersion`.
-
-   :::info
-   Gravitino Spark connector doesn't support Scala 2.13 for Spark3.3.
-   :::
+   under the `spark-connector/spark-connector-runtime/build/libs` directory.
```

Comment:
I think the change here and above is not correct, you'd better revert back to the original one.